### DISCUSSION TITLE
Recording audio is garbled and distorted

### DISCUSSION THREAD

**Discussion by yasmagic** [mynaparrot/plugNmeet-server#619] [General]
Local recording to WEBM file has no audio. Server recording as MP4 file has distorted and missing audio. Is there anything additional needs to be done, like pulseaudio settings? I tried VP8, VP9 and H264, all produce same result.

**Comment by jibon57** [comment]
Recording server should need to be powerful enough otherwise quality of recording will be bad.

**Reply by yasmagic** [reply]
I tested this in 2 servers both are 40 core servers with 64 GB memory. I installed Ubuntu 22 and ran the install script and tested it. It is the same issue, no audio in WEBM local recording and server recording has heavily distorted and missing audio. I also tested with USB microphone and directly on Macbook mic, but I don't think it has to do with client devices.

Does the headless Chromium require anything like pulseaudio or other special tools that requirement additional configuration for Puppeteer?
